src/integration-test/java/org/spongepowered/mod/test/integration/regression/schematic/PlacingSchematicTest.java

```diff
@@ -0,0 +1,161 @@
+            final World schematica = Sponge.getServer().loadWorld(properties).get();
+            final Player player = this.testUtils.getThePlayer();
+            player.offer(Keys.GAME_MODE, GameModes.CREATIVE);
+            player.offer(Keys.IS_FLYING, true);
+            player.setLocation(new Location<>(schematica, 0, 64, 0));
+
+            File inputFile = new File(this.getClass().getClassLoader().getResource("placement.schematic").getFile());
+            DataContainer container = DataFormats.NBT.readFrom(new GZIPInputStream(new FileInputStream(inputFile)));
+            final Schematic toPlace = DataTranslators.SCHEMATIC.translate(container);
+            final Location<World> location = player.getLocation();
+            toPlace.apply(location, BlockChangeFlags.NONE);
+
+            final Vector3i blockMin = toPlace.getBlockMin();
+            final Vector3i blockMax = toPlace.getBlockMax();
+
+            final Vector3i origin = location.getBlockPosition();
+
+            // A note about this:
+            /*
+            Because the deep equals will perform equality checks even
+            on the backing volumes (ArrayBlockVolume vs ArchetypeVolume),
+            the equalities on the directly recreated will fail. So,
+            we send the recreated through the translator back and forth
+            to get a similarly constructed Schematic using an array block volume
+            to perform the equals.
+             */
+            final ArchetypeVolume archetypeVolume = schematica.createArchetypeVolume(origin.add(blockMin), origin.add(blockMax), origin);
+            final Schematic recreated = Schematic.builder()
+                .volume(archetypeVolume)
+                .metaValue(Schematic.METADATA_AUTHOR, "gabizou")
+                .metaValue(Schematic.METADATA_NAME, "placement")
+                .blockPaletteType(PaletteTypes.LOCAL_BLOCKS)
+                .build();
+            final Schematic deserialized = DataTranslators.SCHEMATIC.translate(DataTranslators.SCHEMATIC.translate(recreated));
+            /*
+            This assertion proves the following:
+            - Deserializing a schematic and pasting it into the world works
+            - Creating a new schematic with the *SAME* information as the original schematic (metadata) at the location
+              and then "translating" and "re-translating back" into a Schematic will yeild the an equal Schematic.
+            - The full circle of creating, serializing, deserializing, and translating produces the same
+              schematic as if you loaded a new schematic from file.
+            - Schematics that contain extra data attached to tile entities are retained, but the engine
+              may not respect those when a new schematic is created from the placed original schematic
+             */
+            assertEquals(toPlace, deserialized);
+            return schematica;
+        });
+        this.testUtils.waitForWorldChunks();
```

Comment:
@Aaron1011 this is where I'm kinda struggling to figure out what to do next for future integration tests. I want to be able to see where the schematic ends up, and maybe start building a "redstone" test to trace the `ChangeBlockEvent`s etc.